tests/test_schedule_stream.py

```diff
@@ -1,9 +1,11 @@
 import heterocl as hcl
+import numpy as np
 from itertools import permutations
+import os
```

Comment:
Can you comment out the tests if they are not working right now? It seems some of them can only generate IR? That does not count as *passed*. We need to make sure it is actually executable. At least for software simulation.

Reply:
Do we have to run simulation for the test cases in `tests/test_schedule_stream.py`? I will check the correctness of the generated IR. There might be cases where the IR generated is wrong, but execution result is correct.

Reply:
In this case we need to check both. It's ok we do not check it on CI because there we don't have the needed software, but we need to run the simulation locally to ensure the correctness. Maybe the IR that you think is correct may not have the correct results.